contracts/SpokePool.sol

```diff
@@ -1330,10 +1341,6 @@ abstract contract SpokePool is
         // Verify depositor is a valid EVM address.
         params.depositor.checkAddress();
 
-        // Check that deposit route is enabled for the input token. There are no checks required for the output token
-        // which is pulled from the relayer at fill time and passed through this contract atomically to the recipient.
-        if (!enabledDepositRoutes[params.inputToken.toAddress()][params.destinationChainId]) revert DisabledRoute();
```

Comment:
Second open question: Should we remove this check in the Solana SpokePool as well? I assume no, its not needed for now.

Reply:
Yes, but we'll do it in a separate PR.